contracts/PushOracles/SignedPushOracle.sol

```diff
@@ -19,8 +19,8 @@ contract SignedPushOracle is SignedOracle, PushOracleBase {
     address _dataSource,
     IOracleConsumer _consumer
   )
-    isInitializer("SignedPushOracle", "0.0.0")
     public
+    initializer
```

Comment:
small nit; style wise would be good to keep this before the `public` modifier

Reply:
I _think_ visibility usually comes first but since it looks like that's not consistent with the rest of this code base I'll switch it back and if we decide that's the way to go, we can do a sweep through the code later.